crates/swc_core/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+#![cfg_attr(docsrs, feature(doc_cfg))]
+
+// Quote
+#[cfg(any(docsrs, feature = "quote"))]
```

Comment:
We also need `cfg_attr`

Reply:
is this (https://github.com/swc-project/swc/pull/5364/files#diff-640c564973380a758182cae3dc7d5ea4e31bc82aba03d3b3dfb383273bf93f1dR5) right? and should apply to other places if it's correct?

Reply:
Yes, you can verify by using `cargo doc`
(We specifies `--cfg docsrs` for rustdoc using cargo config toml file)